### DISCUSSION TITLE
Device recipes window UI

### DISCUSSION THREAD

**Discussion by Kpa6i** [voxeltycoon/issues#402] [UI&UX]
Add tooltip with `Item name` when you hover mouse over item. With many new items from different mods it's hard to detect what exactly need to craft a new item.

Currently when you hover a recipe, the recipe frame just slightly increased.

Also, will be helpful to show craft `Duration` time over/under the "arrow" thign.

**Comment by victorleblais** [comment]
I would also add that it should indicate what factory is used.

![template (1)](https://user-images.githubusercontent.com/4365075/65132063-9f5c6b00-da00-11e9-8181-8aa80f20299f.png)
